files/drachtio/drachtio.conf.xml

```diff
@@ -0,0 +1,200 @@
+            <!-- local sip address to bind to.  Default: 'sip:*', which means listens on port 5060 on all interfaces and transports -->
+            <!--
+                    Other examples:
+                        sip:192.168.1.100        listen on a specific address
+                        sip:*;transport=tcp      tcp connections only
+                        sip:*:5061               listen on port 5061
+            -->
+
+
+            <!-- here's how to listen for every protocol on their default ports -->
+<!--
+            <contact>sip:*:80;transport=ws</contact>
+            <contact>sips:*;transport=tls</contact>
+            <contact>sips:*:443;transport=wss</contact>
+-->
+
+        </contacts>
+
+        <!-- uncomment and modify these if you want to change the sip stack timers (globally)
+            <timers>
+                <t1>500</t1>
+                <t2>4000</t2>
+                <t4>5000</t4>
+                <t1x64>32000</t1x64>
+            </timers>
+        -->
+
+        <!-- uncomment this if you send encapsulated SIP messages to homer
+        <capture-server port="9060" hep-version="3" id="101">127.0.0.1</capture-server>
+        -->
+
+        <!-- if you want to terminate SIP over TLS connections (example shows letsencrypt.org typical file locations, but use any CA)
+        <tls>
+                <key-file>/etc/letsencrypt/live/yourdomain/privkey.pem</key-file>
+                <cert-file>/etc/letsencrypt/live/yourdomain/cert.pem</cert-file>
+                <chain-file>/etc/letsencrypt/live/yourdomain/chain.pem</chain-file>
+                <dh-param>/var/local/dh4096.pem</dh-param>
+        </tls>
+        -->
+
+        <!-- if you want to increase the default mtu size for udp packets; stack will force tcp when sending larger packets
+        <mtu-size>4096</mtu-size>
+        -->
+        <udp-mtu>8192</udp-mtu>
+
+        <!-- uncommenting this will cause all outbound new requests (i.e. requests outside of a dialog) to go through the specified proxy -->
+        <!--
+        <outbound-proxy>sip:10.10.10.1</outbound-proxy>
+        -->
+
+        <!-- optional list of header values which, if encountered in a message will trigger an action
+             [tcp-]action="reject|discard"
+             reject means send a sip non-success response immediately
+             discard means sliently discard the request, generating no response
+        -->
+        <spammers action="reject" tcp-action="discard">
+            <header name="User-Agent">
+                <value>sip-cli</value>
+                <value>sipcli</value>
+                <value>friendly-scanner</value>
+            </header>
+            <header name="To">
+                <value>sipvicious</value>
+            </header>
+        </spammers>
+    </sip>
+
+    <!-- set to true if you want the server to cdr events to a connected client -->
+    <cdrs>false</cdrs>
+
+    <monitoring>
+        <prometheus port="8088">127.0.0.1</prometheus>
```

Comment:
This should also be configured to work with the prometheus configured on the existing `compose.monitoring.yaml`